src/main/io/osd.c

```diff
@@ -1203,8 +1204,15 @@ int16_t osdGetHeading(void)
 int16_t osdGetPanServoOffset(void)
 {
     int8_t servoIndex = osdConfig()->pan_servo_index;
-    int16_t servoPosition = servo[servoIndex];
     int16_t servoMiddle = servoParams(servoIndex)->middle;
+    int16_t servoPosition = servo[servoIndex];
+
+    gimbalDevice_t *dev = gimbalCommonDevice();
+    if (dev && gimbalCommonIsReady(dev)) {
+        servoPosition = gimbalCommonGetPanPwm(dev);
+        servoMiddle = PWM_RANGE_MIDDLE;
```

Comment:
One question. Shouldn't this use gimbal_pan_trim for the central position?

Reply:
Makes sense. I updated everything to use trim, (except, raw headtracker).

If no valid headtracker is found, behavior is the same as "center lock" as well.

If there is pan trim, osd will reflect what happens for non head tracked modes as well, so you may need to look a bit sideways to center it in osd. (looking the same direction as non head tracked mode).